src/canopy_server/app.py

```diff
@@ -332,7 +332,15 @@ def _init_engines():
         Tokenizer.initialize()
         kb = KnowledgeBase(index_name=index_name)
         context_engine = ContextEngine(knowledge_base=kb)
-        llm = OpenAILLM()
+        if llm == "OpenAILLM":
+            llm = OpenAILLM()
+        elif llm == "AzureOpenAILLM":
+            llm = AzureOpenAILLM(
+                api_key=os.getenv("OPENAI_API_KEY"),
+                base_url=os.getenv("OPENAI_BASE_URL"),
```

Comment:
The original OpenAI client is automatically gettitng the env, isn't the AzureOpenAIClient doing the same? if so we prefer handling it this way

Reply:
seems it does:
https://github.com/openai/openai-python/blob/main/src/openai/lib/azure.py#L160